packages/cozy-sharing/src/components/recipient.styl

```diff
@@ -53,18 +53,9 @@
     flex 1 1 auto
     overflow hidden
     text-overflow ellipsis
-
-.recipient-status
-    flex 0 0 auto
-    font-size .875rem
-    color var(--coolGrey)
-
-.recipient-ident-status
-    flex 1 1 auto
-    display flex
-    flex-direction row
-    align-items center
-    max-width calc(100% - 3rem)
+    
+.recipient-status-icon
+  margin-right: 4px
```

Comment:
Where this value cames from? I don't see it in https://github.com/cozy/cozy-ui/blob/master/stylus/settings/spaces.styl#L14-L21 

Also, should we really use `px`?

Reply:
You're right, i'm changing it to rems. And as other values in this file and design, they don't match up with the spacing scale. That being said, this one in particular is not a traditional margin, it's spacing between and icon and text that needs to be larger than a regular space to look good.